hack/make-bundle-vars.mk

```diff
@@ -65,6 +65,11 @@ NOOBAA_BUNDLE_IMG_TAG ?= v5.9.0
 NOOBAA_BUNDLE_IMG_LOCATION ?= quay.io/noobaa
 NOOBAA_BUNDLE_IMG ?= $(NOOBAA_BUNDLE_IMG_LOCATION)/$(NOOBAA_BUNDLE_IMG_NAME):$(NOOBAA_BUNDLE_IMG_TAG)
 
+CSI_ADDONS_BUNDLE_IMG_TAG ?= v0.1.1
+CSI_ADDONS_BUNDLE_IMG_LOCATION ?= quay.io/csiaddons/k8s-bundle
+CSI_ADDONS_BUNDLE_IMG ?= $(CSI_ADDONS_BUNDLE_IMG_LOCATION):$(CSI_ADDONS_BUNDLE_IMG_TAG)
+CSI_ADDONS_BUNDLE_NAME ?= csi-addons
+
```

Comment:
Can you pls follow the way it is done for ocs and noobaa

Reply:
The problem here is that the bundle name will be different, for the image location we have k8s-bundle but elsewhere we are using bundle name as csi-addons, hence there is a difference.

Reply:
I guess we should rename this `k8s-bundle` image name to `csi-addons-bundle` that is how we are following in all projects. cc @nixpanic

Reply:
@iamniting I guess we discussed this in the meeting and you said it's fine, maybe I misunderstood. But if it's fine can we do this later as we don't have enough time or it is something required?

Reply:
It is not a hard requirement, we can do this later. But I would prefer it to be `csi-addons-bundle` or something which makes more sense as `k8s-bundle` is a little vague.